clab/file.go

```diff
@@ -35,6 +35,10 @@ type TopoFile struct {
 	name     string // file name without extension
 }
 
+func (tf *TopoFile) GetDir() string {
```

Comment:
just curious why you wanted this function, instead of accessing the field?

Reply:
the issue is, that we put applogic in the cmd package ... what we should do is put all the app logic in the clab package and just do parameter parsing and population in the cmd package... that is not too clean of a structure. As a result of this, from the cmd package we are unable to access the lower case dir variable... instead of opting for making the var public (CamelCase) I introduced this getter.
We could come up with a seperate PR and seperate app logic from parameter paring and initialization if you like, I'd be happy to do so.